packages/sanes-chrome-extension/src/routes/requests/index.stories.tsx

```diff
@@ -26,8 +27,8 @@ const intialRequests: Request[] = [
       senderUrl: 'Sender 2',
       requestedIdentities: [],
     },
-    accept: jest.fn(),
```

Comment:
Usage of `jest.fn()` breaks storybook.

Reply:
🤔 I can figure out the reason but the question is... why suddenly the usage of `jest.fn()` breaks storybook and not before? We created this class long time ago.

Any idea?

Reply:
Storybook continue working but some of the stories become unavailable, especially in `Extension` part.